src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -90,30 +95,47 @@ public class SkriptCommand implements CommandExecutor {
 		if (TestMode.DEV_MODE)
 			SKRIPT_COMMAND_HELP.add("test");
 	}
-	
-	private static void reloading(CommandSender sender, String what, Object... args) {
+
+	private static void reloading(CommandSender sender, String what, RedirectingLogHandler logHandler, Object... args) {
 		what = args.length == 0 ? Language.get(CONFIG_NODE + ".reload." + what) : Language.format(CONFIG_NODE + ".reload." + what, args);
-		Skript.info(sender, StringUtils.fixCapitalization(m_reloading.toString(what)));
+		String message = StringUtils.fixCapitalization(m_reloading.toString(what));
+		Skript.info(sender, message);
+
+		// Log reloading message
+		if (SkriptConfig.sendReloadingInfoToOps.value())
+			logHandler.log(new LogEntry(Level.INFO, Language.format(CONFIG_NODE + ".reload." + "player reload", sender.getName(), what)));
 	}
-	
+
+
+
 	private static final ArgsMessage m_reloaded = new ArgsMessage(CONFIG_NODE + ".reload.reloaded");
 	private static final ArgsMessage m_reload_error = new ArgsMessage(CONFIG_NODE + ".reload.error");
-	
-	private static void reloaded(CommandSender sender, RedirectingLogHandler r, TimingLogHandler timingLogHandler, String what, Object... args) {
+
+	private static void reloaded(CommandSender sender, RedirectingLogHandler logHandler, TimingLogHandler timingLogHandler, String what, Object... args) {
 		what = args.length == 0 ? Language.get(CONFIG_NODE + ".reload." + what) : PluralizingArgsMessage.format(Language.format(CONFIG_NODE + ".reload." + what, args));
-		String timeTaken  = String.valueOf(timingLogHandler.getTimeTaken());
+		String timeTaken = String.valueOf(timingLogHandler.getTimeTaken());
+
+		String message;
+		if (logHandler.numErrors() == 0) {
+			message = StringUtils.fixCapitalization(PluralizingArgsMessage.format(m_reloaded.toString(what, timeTaken)));
+			Skript.info(sender, message);
+		} else {
+			message = StringUtils.fixCapitalization(PluralizingArgsMessage.format(m_reload_error.toString(what, logHandler.numErrors(), timeTaken)));
+			Skript.error(sender, message);
+		}
 
-		if (r.numErrors() == 0)
-			Skript.info(sender, StringUtils.fixCapitalization(PluralizingArgsMessage.format(m_reloaded.toString(what, timeTaken))));
-		else
-			Skript.error(sender, StringUtils.fixCapitalization(PluralizingArgsMessage.format(m_reload_error.toString(what, r.numErrors(), timeTaken))));
+		// Notify operators
+		if (SkriptConfig.sendReloadingInfoToOps.value())
+			logHandler.log(new LogEntry( Level.INFO, message));
```

Comment:
This would send to the `sender` twice. It would be better to do something like:
```
if no errors:
  logHandler.log(message) // log with INFO
else:
  logHandler.log(message) // log with SEVERE
```
the log handler will already handle sending to operators so you don't need to check it again